app/controllers/welcome_controller.rb

```diff
@@ -1,3 +1,9 @@
 class WelcomeController < ApplicationController
-  def index; end
+  def show
+    @image = Image.find(params[:id])
+  end
+
+  def index
+    @images = Image.order('created_at DESC')
```

Comment:
After we move this to ImagesController, let's make sure to add tests for it. Try to come up with a few test cases on your own based on the previous tests you've written (ie. success, failure, record not found, no existing images, etc.) and the functionality required in this story (ie. image order)